src/html/index.html.template

```diff
@@ -9,6 +9,7 @@
     <link rel='apple-touch-icon' sizes='180x180'
           href='/static/icons/favicon-apple-180x180.png'>
     <link rel="mask-icon" href="/static/icons/mask-icon.svg" color="#03a9f4">
+    <meta name="apple-itunes-app" content="app-id=1099568401">
```

Comment:
Is this bound to a specific domain name, or is just this tag enough?

Reply:
Just this tag is enough.